bentoml/yatai/yatai_service.py

```diff
@@ -123,6 +134,12 @@ def start_yatai_service_grpc_server(
                 '"pip install grpcio-reflection"'
             )
     server.add_insecure_port(f'[::]:{grpc_port}')
+    # TODO: it seems that current impl doesn't actually intercept RPC calls
+    prometheus_port = 50052
+    with reserve_free_port() as port:
```

Comment:
what is this trying to archive?

Reply:
When I had this conversation with bozhao, the expected behaviour is to use `reserve_free_port` to allocate a port for prometheus internally. However, it seems to be that in most cases `50052` is free so I just put it as fallback when others want to run `make run-yatai-web-ui`. If possible I think we should allocate `50052` to prometheus permanently.